handlers/bounty.go

```diff
@@ -19,14 +19,16 @@ import (
 )
 
 type bountyHandler struct {
-	httpClient HttpClient
-	db         db.Database
+	httpClient            HttpClient
+	db                    db.Database
+	generateBountyHandler func(bounties []db.Bounty) []db.BountyResponse
```

Comment:
@AbdulWahab3181 change this to `generateResponse`,  `generateBountyHandler`, sounds a bit misleading.

Reply:
Addressed